db/migrate/20220413213421_add_domain_to_organisation.rb

```diff
@@ -0,0 +1,6 @@
+class AddDomainToOrganisation < ActiveRecord::Migration[6.0]
```

Comment:
I have seen AAF committing the magic string:
`# frozen_string_literal: true`
as the first line of all Ruby code, including migrations - I expect Rubocop will complain if it isn't there...

Reply:
No longer required by Rubocop.